ocs_ci/helpers/dr_helpers.py

```diff
@@ -1083,3 +1098,258 @@ def verify_drpolicy_cli(switch_ctx=None):
         raise UnexpectedBehaviour(
             f"DRPolicy is not in succeeded or validated state: {status}"
         )
+
+
+def enable_cluster_backup():
+    """
+    set cluster-backup to True in mch resource
+    Note: changing this flag automatically installs OADP operator
+    """
+    mch_resource = ocp.OCP(
+        kind=constants.ACM_MULTICLUSTER_HUB,
+        resource_name=constants.ACM_MULTICLUSTER_RESOURCE,
+        namespace=constants.ACM_HUB_NAMESPACE,
+    )
+    mch_resource._has_phase = True
+    resource_dict = mch_resource.get()
+    for components in resource_dict["spec"]["overrides"]["components"]:
+        if components["name"] == "cluster-backup":
+            components["enabled"] = True
+    mch_resource_yaml = tempfile.NamedTemporaryFile(
+        mode="w+", prefix="mch", delete=False
+    )
+    yaml_serialized = yaml.dump(resource_dict)
+    mch_resource_yaml.write(yaml_serialized)
+    mch_resource_yaml.flush()
+    run_cmd(f"oc apply -f {mch_resource_yaml.name}")
+    mch_resource.wait_for_phase("Running")
+    backup_pod_status_check()
+
+
+def create_s3_bucket(access_key, secret_key, bucket_name):
```

Comment:
In automation do we use s3 by default and not noobaa ?

Reply:
Yes we use s3 coming from aws